R/plotly_build.R

```diff
@@ -394,7 +394,7 @@ registerFrames <- function(p, frameMapping = NULL) {
     # retrain colors on each frame (including other data that isn't animated)
     frameDat <- retrain_color_defaults(p$x$data[thisFrame | isNotAFrame])
     p$x$frames[[i]] <- list(
-      name = frameNames[i],
+      name = as.character(frameNames[i]),
```

Comment:
I'm just glancing through this for workaround that have since been fixed. This is now applied in plotly.js at https://github.com/plotly/plotly.js/pull/1124